src/fiatconnect/index.ts

```diff
@@ -29,6 +47,43 @@ export async function getFiatConnectProviders(): Promise<FiatConnectClientConfig
   return providers
 }
 
+type QuotesInput = QuoteRequestQuery & {
+  fiatConnectProviders: FiatConnectProviderInfo[]
+  flow: CICOFlow
+}
+
+type GetFiatConnectQuotesResponse = {
+  quotes: (Result<QuoteResponse, QuoteErrorResponse | { error: string }> & { id: string })[]
+}
+
+export async function getFiatConnectQuotes(params: QuotesInput): Promise<FiatConnectQuote[]> {
+  const { flow, fiatConnectProviders, ...otherParams } = params
+  const cleanParams = pickBy(otherParams, (v) => v !== undefined)
+  const providers = fiatConnectProviders.map((provider) => provider.id).join(',')
+  const queryParams = new URLSearchParams({
+    ...(cleanParams as Record<string, string>),
+    providers,
+    quoteType: flow === CICOFlow.CashIn ? 'in' : 'out',
+  }).toString()
+  const response = await fetch(`${networkConfig.getFiatConnectQuotesUrl}?${queryParams}`)
+  if (!response.ok) {
+    const err = await response.json()
+    Logger.error(TAG, `Failure response fetching FiatConnect quotes: ${err} , returning empty list`)
+    return []
+  }
+  const results: GetFiatConnectQuotesResponse = await response.json()
```

Comment:
if the response came back with an error, it should be included in the `!response.ok` case? why does `GetFiatConnectQuotesResponse` type need to include `{ error: string}` ?